jrcc-access-spring-boot-autoconfigure/src/main/java/ca/bc/gov/open/jrccaccess/autoconfigure/plugins/rabbitmq/RabbitMqDocumentInput.java

```diff
@@ -60,6 +60,10 @@ public void receiveMessage(DocumentReadyMessage documentReadyMessage) throws Doc
 		}
 		
 		this.documentReadyHandler.handle(content, documentReadyMessage.getTransactionInfo());
+
+                //when successfully processed, delete the document from redis storage
+                this.redisStorageService.deleteString(storageProperties.getKey());
```

Comment:
can you add the logger here, 
* before `debug` -> attempting to ...
* after `info` -> document deleted ...